examples/real-world/store/configureStore.prod.js

```diff
@@ -1,14 +1,11 @@
 import { createStore, applyMiddleware } from 'redux'
 import createSagaMiddleware, { END } from 'redux-saga'
+import sagaMonitor from 'saga-monitor-example'
```

Comment:
Should we change configureStore.dev.js as well?

Reply:
for sure, it should.
now when I add it I can see that `saga-monitor-example` is not compiled and not ready to be consumed. need to figure out how to fix it

Reply:
this is somewhat a problem because this example is marked as private now - we can add a build step & publish it, that way codesandbox will know how to consume it

Reply:
shouldn't we switch to https://github.com/redux-saga/redux-saga-devtools package?

Reply:
yeah, that would be nice too - i dont know how up to date it is though & don't have time personally to adjust those examples to use it, publishing saga-monitor-example has just seemed like smaller effort